sherlock/resources/data.json

```diff
@@ -1898,6 +1914,13 @@
     "urlProbe": "https://ch.tetr.io/api/users/{}",
     "username_claimed": "osk"
   },
+  "Tilde.club": {
```

Comment:
Neocities has info on valid usernames. With this, we can craft a regex:

```
^[a-zA-Z0-9]+(-[a-zA-Z0-9]+)*$
```

Could you include this as well?